### PR TITLE
Dockerize the website

### PR DESCRIPTION
**Pull request by zorexsalvo**
Fixes issue #96 
---
Changes:
- Added  Dockerfile and docker-compose.yml
- Changed database engine, used postgreSQL instead of sqlite
- Fix the failing travis build

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.6.2
+LABEL maintainer="Programming club, UIET, Panjab University"
+
+COPY requirements.txt /opt/
+RUN pip install -r /opt/requirements.txt
+
+COPY . /opt/
+WORKDIR /opt/
```
Comment (divyam3897) [inline Dockerfile]: change it to a better name and not /opt which should signify what it contains.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+website:
+  build: .
+  volumes:
+    - .:/opt/
+  ports:
+    - 8000:8000
```
Comment (divyam3897) [inline docker-compose.yml]: Add db settings also

Comment (zorexsalvo) [reply docker-compose.yml]: Sure, but what DB would you like me to add, PostgreSQL? Right now the project is using sqlite.

**Comment by zorexsalvo** [comment]
@divyam3897 Okay thanks! Will apply the changes in a day.

**Comment by divyam3897** [comment]
are you still working on it @zorexsalvo ?

**Comment by zorexsalvo** [comment]
@divyam3897 Yes sorry, if there still isn't working on it. I just can't make a time for it these past days.

**Comment by divyam3897** [comment]
@zorexsalvo still have time for it or I can close this one?

**Comment by zorexsalvo** [comment]
@divyam3897 Ill give it a shot today. Thank you for your patience.

**Comment by zorexsalvo** [comment]
@divyam3897 Oh no, the travis-ci is failing. Must change the travis installation config.

**Comment by divyam3897** [comment]
It has been merged thank you :)

**Comment by zorexsalvo** [comment]
@divyam3897 No problem! :100:  Thanks too :+1:

**Pull request merged**